bundles/framework/maplegend/MapLegendList.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Collapse } from 'antd';
```

Comment:
Is there a reason to use antd directly instead of the one from oskariui? You wouldn't have to import the style here if you import from oskariui.

Reply:
Fixed.